deployment/helm/skaha/sample-local-values.yaml

```diff
@@ -35,6 +32,18 @@ deployment:
       minEphemeralStorage: "10Gi" # Initial request for ephemeral storage (scratch disk space)
       maxEphemeralStorage: "100Gi" # Max expansion for ephemeral storage (scratch disk space)
 
+      # When set to 'true' this flag will enable GPU node scheduling.
+      gpuEnabled: true
+
+      # Example nodeAffinity to restrict how User Sessions are scheduled.
```

Comment:
I don't think we need a generalized scheduling of user jobs on certain nodes.  We do need however, generalized scheduling of system deployments on certain nodes.  That means that we need generalized avoidance of user jobs on those nodes.